### ISSUE TITLE
Settings/Admin Dashboard Page

### ISSUE THREAD

**Issue by innopoop** [novaforgood/RISC-Frontend#30]
[figma here](https://www.figma.com/file/qfzd9rev3yhzKeCZBv8lIB/wireframe?node-id=3090%3A3612)
Deadline: 6/4

Blocked by:
1. Textarea #1 
2. Mini Button
3. Pulling program information & Storing it locally/caching [may use dummy information for now]

Things to get done
1. Admin Preview Section
2. Altering program information (using mutations)
3. Styling
4. [OPTIONAL (if not done here create a new issue)] Add administrators to a program
